Assets/Scripts/Model/Unit.cs

```diff
@@ -22,6 +22,7 @@ public class Unit : MonoBehaviour, CanReceiveDamage, HUDSubject
     public Texture damagedTexture;
```

Comment:
Since you're changing the other thing do

git checkout dev -- Assets/Scripts/Models/Unit.cs

this will return Unit.cs to its dev state and won't push unnecessary changes